.azure-pipelines/jobs/test-ubuntu.yml

```diff
@@ -24,11 +24,12 @@ jobs:
         displayName: "Run tests"
         env:
           TF_ACC: 1
-          CTP_AUTH_URL: http://localhost:8989
-          CTP_PROJECT_KEY: unittest
+          CTP_CLIENT_ID: clientid
+          CTP_CLIENT_SECRET: clientsecret
+          CTP_PROJECT_KEY: projectkey
+          CTP_SCOPES: view_project:projectkey
```

Comment:
I still think that the scopes must be provided, there should not be a default to `manage_project`. Can someone confirm that this is ok to change?

Reply:
yes, if it's not recommended for production we should indeed not offer it default like this.